src/Contexts/User-Context/user-reducer.js

```diff
@@ -0,0 +1,36 @@
+const userData = JSON.parse(localStorage.getItem("authToken"));
```

Comment:
suggestion: you can use function for token and use it everywhere.

Reply:
Yeah done that